oval/types.go

```diff
@@ -89,16 +89,26 @@ type Reference struct {
 // Advisory : >definitions>definition>metadata>advisory
 // RedHat and Ubuntu OVAL
 type Advisory struct {
-	XMLName         xml.Name   `xml:"advisory"`
-	Severity        string     `xml:"severity"`
-	Cves            []Cve      `xml:"cve"`
-	Bugzillas       []Bugzilla `xml:"bugzilla"`
-	AffectedCPEList []string   `xml:"affected_cpe_list>cpe"`
-	Refs            []Ref      `xml:"ref"` // Ubuntu Only
-	Bugs            []Bug      `xml:"bug"` // Ubuntu Only
-	PublicDate      Date       `xml:"public_date"`
-	Issued          Date       `xml:"issued"`
-	Updated         Date       `xml:"updated"`
+	XMLName         xml.Name         `xml:"advisory"`
+	Severity        string           `xml:"severity"`
+	Cves            []Cve            `xml:"cve"`
+	Bugzillas       []Bugzilla       `xml:"bugzilla"`
+	AffectedCPEList []string         `xml:"affected_cpe_list>cpe"`
+	Refs            []Ref            `xml:"ref"` // Ubuntu Only
+	Bugs            []Bug            `xml:"bug"` // Ubuntu Only
+	PublicDate      Date             `xml:"public_date"`
+	Issued          Date             `xml:"issued"`
+	Updated         Date             `xml:"updated"`
+	Affected        []AffectedStatus `xml:"affected"` // Red Hat Only
+}
+
+type AffectedStatus struct {
+	Resolution []Resolution `xml:"resolution"`
```

Comment:
Can there be multiple `resolution` objects?

Reply:
nit: `Resolutions`

@hdonnay, yes, for example CVE-2019-25067:

```
<affected>
    <resolution state="Affected">
     <component>container-tools:rhel8/podman</component>
     <component>container-tools:rhel8/podman-catatonit</component>
     <component>container-tools:rhel8/podman-debugsource</component>
     <component>container-tools:rhel8/podman-docker</component>
     <component>container-tools:rhel8/podman-gvproxy</component>
     <component>container-tools:rhel8/podman-manpages</component>
     <component>container-tools:rhel8/podman-plugins</component>
     <component>container-tools:rhel8/podman-remote</component>
     <component>container-tools:rhel8/podman-tests</component>
    </resolution>
    <resolution state="Out of support scope">
     <component>container-tools:2.0/podman</component>
     <component>container-tools:2.0/podman-catatonit</component>
     <component>container-tools:2.0/podman-debugsource</component>
     <component>container-tools:2.0/podman-docker</component>
     <component>container-tools:2.0/podman-gvproxy</component>
     <component>container-tools:2.0/podman-manpages</component>
     <component>container-tools:2.0/podman-plugins</component>
     <component>container-tools:2.0/podman-remote</component>
     <component>container-tools:2.0/podman-tests</component>
    </resolution>
   </affected>
```